test/e2e/helm_chart_test.go

```diff
@@ -62,16 +62,29 @@ var _ = Describe("HELM charts", func() {
 	})
 
 	DescribeTable("Namespaced operators working only with its own namespace with different configuration", Label("helm-ns"),
-		func(test model.TestDataProvider) {
+		func(test model.TestDataProvider, cType string) { // cType - probably will be moved later ()
```

Comment:
I think there is no harm in renaming `cType` into `clusterType` cause its type isn't descriptive)